proposals/zips/zip001.sol

```diff
@@ -58,16 +74,12 @@ contract zip001 is Proposal {
         /// Set MINTER role for all NFT minting contracts
         _core.grantRole(Roles.MINTER_PROTOCOL_ROLE, addresses.getAddress("ERC1155_MAX_SUPPLY_MINTABLE_WEARABLES"));
         _core.grantRole(Roles.MINTER_PROTOCOL_ROLE, addresses.getAddress("ERC1155_MAX_SUPPLY_ADMIN_MINTER"));
-    }
-
-    function _aferDeployForTestingOnly(Addresses, address deployer) internal virtual override {}
 
-    function _afterDeployOnChain(Addresses, address deployer) internal override {
-        // Revoke ADMIN role from deployer
-        _core.revokeRole(Roles.ADMIN, deployer);
+        console.log("Revoke ADMIN role from deployer on mainnet");
+        // _core.revokeRole(Roles.ADMIN, deployer);
```

Comment:
👀

Reply:
Earlier there were two functions `_aferDeployForTestingOnly` and `_afterDeployOnChain`. This was done to revoke deployer admin permission on mainnet only and not for any of the testnets. So now as this zip has been deployed already on mainnet we don't need to keep these two separate functions. By commenting this action I am able to have a single `_afterDeploy`.